maskrcnn_benchmark/structures/segmentation_mask.py

```diff
@@ -1,51 +1,93 @@
 # Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
 import torch
-
+import numpy as np
+from torch.nn.functional import interpolate
 import pycocotools.mask as mask_utils
 
 # transpose
 FLIP_LEFT_RIGHT = 0
 FLIP_TOP_BOTTOM = 1
 
-
 class Mask(object):
     """
     This class is unfinished and not meant for use yet
     It is supposed to contain the mask for an object as
     a 2d tensor
     """
-
-    def __init__(self, masks, size, mode):
-        self.masks = masks
+    def __init__(self, segm, size, mode):
+        width, height = size
+        if isinstance(segm, Mask):
+            mask = segm.mask
+        else:
+            if type(segm) == list:
+                # polygons
+                rle = mask_utils.frPyObjects(segm, height, width)
+                mask = np.array(mask_utils.decode(rle), dtype=np.float32)
+                mask = np.sum(mask, axis=2)
```

Comment:
Is this sum here needed?

Reply:
I think it is need according to the code in Detectron and my experience. A binary mask may contain multiple polygons and to convert polygons into binary mask, the third dimension should be merged by summation.

Reply:
`sum` here is equivalent to `merge` in `Pylygons.convert`.